specs/UI_import_gluster_cluster_workflow.adoc

```diff
@@ -0,0 +1,218 @@
+:imagesdir: ./images/
+
+= Import gluster cluster workflow
+
+== Problem description
+
+This specification describes import gluster cluster workflow which is a mechanism to add an gluster storage cluster into tendrl for management.
+
+View design will look like:
+
+image::import.png[Import workflow]
+
+== Use Cases
+
+This addresses the use case of importing existing gluster clusters into tendrl for management.
+
+The workflow is as follows:
+
+* API will provide the list of clusters(imported and ready to import) and those clusters will be displayed on cluster list page.
+
+* A button will displayed for each cluster which are not imported and are ready to import.
+
+* User will click on the import button for the cluster, he wants to import.
+
+* A popup will be opened for it.
+
+* UI will filter the available nodes for importing the cluster.
+
+* The pop will list down the available nodes and a checkbox for enabling/disabling the cluster wide volume profiling.
+
+* Each node's name, IP address and role will be shown.
```

Comment:
@brainfunked @nthomas-redhat @julienlim Earlier we were displaying node's name, release and role for each node. And now as per the updated mockup [https://redhat.invisionapp.com/share/8QCOEVEY9#/screens/247445416](url) its node's name and IP address. Need confirmation on the shown fields for each node.